third-party/rspamd/src/test/java/org/apache/james/rspamd/client/RSpamDHttpClientTest.java

```diff
@@ -99,11 +99,6 @@ void checkSpamMailUsingRSpamDClientWithExactPasswordShouldReturnAnalysisResultAs
         RSpamDHttpClient client = new RSpamDHttpClient(configuration);
 
         AnalysisResult analysisResult = client.checkV2(spamMessage).block();
-        assertThat(analysisResult).isEqualTo(AnalysisResult.builder()
-            .action(AnalysisResult.Action.REJECT)
```

Comment:
Can we still make sure that it's detected as a spam an rejected? No need to check the rest